README.md

```diff
@@ -41,26 +41,32 @@ You can specify the following options in the `.testcafe-electron-rc` configurati
 
 ### mainWindowUrl
 
-__Required.__ Specifies the URL of the application's main window page. 
+__Required.__ Specifies the URL of the application's main window page.
  For local application pages, you can also specify a relative (to the application directory) or an absolute path to the file of the page.
 
 ### appPath
 
-__Optional.__ Alters path to the application. By default, the application path is specified after the `electron:` 
- prefix in the string passed to the command line or API. You can use the `appPath` option to override it by specifying an absolute path, or append a relative path to it.  
+__Optional.__ Alters path to the application, which, by default, must be located at the [default Electron app directory](https://github.com/electron/electron/blob/master/docs/tutorial/application-distribution.md#application-distribution).
+ You can use the `appPath` option to override it by specifying an absolute path, or append a relative path to the path specified after the `electron:` prefix
```

Comment:
This sentence is very complicated for me. @VasilyStrelyaev provided following variant:
You can use the `appPath` option to override the default path by specifying a new absolute path. Alternatively, you can append a relative path to the path specified after the `electron:` prefix.